download_dicts.sh

```diff
@@ -1,2 +1,57 @@
-wget http://download.wikdict.com/dictionaries/sqlite/1/all.tar.gz -O data/all.tar.gz
-mkdir -p data/dict && tar xvfz data/all.tar.gz --directory data/dict && rm data/all.tar.gz
+#! /usr/bin/env bash
+# Copyright (c) 2021 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license, see file COPYING for the full text
+
+set -x
+set -e
+
+languages=(
+    bg
+    de
+    el
+    en
+    es
+    fi
+    fr
+    id
+    it
+    ja
+    la
+    lt
+    mg
+    nl
+    no
+    pl
+    pt
+    ru
+    sv
+    tr
+)
+
+language_pairs=()
+for language_from in "${languages[@]}"; do
+    for language_to in "${languages[@]}"; do
+        if [[ "${language_from}" = "${language_to}" ]]; then
+            continue
+        fi
+
+        language_pairs+=( "${language_from}-${language_to}" )
+    done
+done
+
+
+mkdir -p data/dict
+
+find data/dict/ -type f -name '*.sqlite3' -empty -print -delete
```

Comment:
This one is quite destructive. Since we overwrite the files anyway, couldn't we leave this out? I'm a bit uneasy about deleting with wildcards without asking for permission.

Reply:
It's only zero-byte files (which cause trouble) but sure, I'll drip that line :+1:

Reply:
Gone